test_runner.py

```diff
@@ -118,15 +137,5 @@ def run_test(test_flavor: OverrideDefinitions, full_path: str):
             config = tomllib.load(f)
             is_integration_test = config["job"].get("use_for_integration_test", False)
             if is_integration_test:
-                test_flavors = [OverrideDefinitions()] + integration_tests_flavors[
```

Comment:
OverrideDefinitions() from here got move to the top as 'Default' entry, but specified its own output dir.